### ISSUE TITLE
[Accessibility]A11y_VSCode_WelcomeScreen_ScreenReader: VoiceOver reads some irrelevant content and does not read the information of lightbox on hitting "Interface Overview" button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52435]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader : VoiceOver

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Navigate to Welcome screen.
3. Navigate to "Interface Overview" button and hit "Enter".
4. Try to read the content that pops up.

**Actual:**
VoiceOver reads some irrelevant content and does not read the information of lightbox. It reads the "Interactive Playground" button,information of "Activity" bar buttons and reads information from page behind.

**Expected:**
VoiceOver should only read the content of the lightbox once it appears. Irrelevant content should not be read before reading the content.

**Recommendations:**
Provide tabindex to the entire lightbox that has appeared and give title to it. Then the content of the lightbox should be read.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not know the exact content that has been presented using lightbox and also user will not know on which screen he is on.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_WelcomeScreen_VoiceOver_LightBoxIssue.pptx](https://github.com/Microsoft/vscode/files/2118809/A11y_VSCode_WelcomeScreen_VoiceOver_LightBoxIssue.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51983
